index.html

```diff
@@ -0,0 +1,217 @@
+        </ul>
+    </div>
+    <div class="container">
+    <div class="navbar">
+        <div class="logo">
+            <a href="index.html"><img src="./images/newlogo.png" alt="African Tech Summit Logo"></a>
+        </div>
+        <nav>
+            <ul>
+                <li><a href="about.html">About</a></li>
+                <li><a href="#">Program</a></li>
+                <li><a href="#">Join</a></li>
+                <li><a href="#">Sponsor</a></li>
+                <li><a href="#">News</a></li>
+                <li class="border-about"><a href="#">CC Campaign</a></li>
+            </ul>
+        </nav>
+    </div>
+    </div>
+        <div class="bars" id="hambug">
+        <div class="hambuger"></div>
+        <div class="hambuger"></div>
+        <div class="hambuger"></div>
+        </div>
+
+    <div class="toggle-menu">
+            <div><img src="./images/closetoggle.png" alt="close toggle" id="togclose" ></div>
+
+            <div class="hambug-items">
+            <ul class="menu">
+                <li class="list-menu"><a href="about.html" class="a-link">About</a></li>
+                <li class="list-menu"><a href="#" class="a-link">Program</a></li>
+                <li class="list-menu"><a href="#" class="a-link">Join</a></li>
+                <li class="list-menu"><a href="#" class="a-link">Sponsor</a></li>
+                <li class="list-menu"><a href="#" class="a-link">News</a></li>
+                <li class="list-menu"><a href="#" class="a-link">CC Campaign</a></li>
+            </ul>
+            </div>
+
+    </div>
+    </header>
+
+    <section class="hero">
+        <div class="container intro">
+        <h1>"Hello! Tech Africa"</h1>
+        <h2>FRESH START <br> AFRICA TECH SUMMIT 2022</h2>
+        <P class="description">The Africa tech summit is an event where leaders from all around Africa connects, explore and invest. Happening on the 5th of April in Nigeria</P>
+        <h3><span>2022.04.05(TUE) - 08(FRI)</span></h3>
+        <P>@ International Conference Center, Garki, Abuja, Nigeria.</P>
+        </div>
+    </section>
+
+    <section class="main-program">
+        <div class="main-head">
+        <h3>Main Program</h3>
+        <hr>
+        </div>
+        <div class="main-blocks">
+            <div class="lecture">
+                <div class="icon-container">
+                <img src="./images/lecture.png" alt="Lecture icon">
+                </div>
+                <h3 class="lecture-title">Lecture</h3>
+                <p class="lecture-desc">We have curated and lined up excellent keynotes from industry leaders.</p>
+
+            </div>
+            <div class="lecture">
+                <div class="icon-container">
+                <img src="./images/CC.png" alt="Exhibition icon">
+                </div>
+                <h3 class="lecture-title">Exhibition</h3>
+                <p class="lecture-desc">innovations from emerging tech hubs across Africa will be showcased.</p>
```

Comment:
The main program section in the mobile version should look almost identical to the [original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015).

Your design:
![image](https://user-images.githubusercontent.com/60292761/148424921-ddb6c703-3116-4241-adb9-b92ad3695e8f.png)

The original design : 
![image](https://user-images.githubusercontent.com/60292761/148425003-b773df18-770a-442c-b3cf-494d20ae4389.png)

Please make sure to follow the original design paddings and margins.

Reply:
I have made the adjustments to this